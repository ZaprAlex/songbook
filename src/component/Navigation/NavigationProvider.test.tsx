import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { NavigationProvider } from '.';

describe('NavigationProvider', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<NavigationProvider />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});
