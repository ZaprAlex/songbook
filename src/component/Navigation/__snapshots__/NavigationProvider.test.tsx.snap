// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationProvider renders correctly 1`] = `
<ContextProvider
  value={
    Object {
      "goBack": [Function],
      "goRoot": [Function],
      "goTo404": [Function],
      "goTo500": [Function],
      "goToChords": [Function],
      "goToCommonError": [Function],
      "goToSongs": [Function],
    }
  }
/>
`;
